colossalai/initialize.py

```diff
@@ -392,7 +396,7 @@ def initialize(model: nn.Module,
         gradient_handlers = [build_gradient_handler(cfg, model, optimizer) for cfg in gradient_handler_cfg]
 
     # check if optimizer is ColossalaiOptimizer
-    if not isinstance(optimizer, (ColossalaiOptimizer, ShardedOptimizer)):
+    if not isinstance(optimizer, (ColossalaiOptimizer, ShardedOptimizerV2)):
         optimizer = ColossalaiOptimizer(optim=optimizer)
 
     # gradient accumulation
```

Comment:
@ver217 @FrankLeeeee 
can ```accumulate_gradient``` work together with zero?

Reply:
I think it is compatible. accumulate_gradient only skips the optimizer.step() call until accumulate_size is reached.

Reply:
cool